requirements.txt

```diff
@@ -39,12 +33,7 @@ eventlet==0.33.3
 factory-boy==3.3.0
 Faker==19.3.0
 ffmpeg-python==0.2.0
-Flask==2.1.3
-Flask-Cors==4.0.0
-Flask-RESTful==0.3.10
-Flask-SocketIO==4.3.2
-flask-talisman==1.1.0
-g2p==1.1.20230511 # https://github.com/roedoejet/g2p
+g2p==1.1.20230822 # https://github.com/roedoejet/g2p
```

Comment:
@bronson-c @caforbes Do we want to upgrade g2p right now?

Reply:
I checked the update, I don't think it will affect anything either way. Maybe if there's a language with `$` or `^` in the alphabet but I don't think there is. It pins a npm dependency (`handsontable@12.4`) but I don't think this will affect us.

Reply:
Thanks!